components/headers/HeaderActionRight.tsx

```diff
@@ -5,17 +5,21 @@ import IconButton from '../buttons/IconButton';
 import Row from '../common/Row';
 
 type HeaderActionRightProps = {
-  collapsed: boolean;
-  onCollapse: () => void;
   handlePress: () => void;
+  toggleView: () => void;
+  listVisible: boolean;
 };
 
-const HeaderActionRight = ({ collapsed, onCollapse, handlePress }: HeaderActionRightProps) => {
+const HeaderActionRight = ({ toggleView, listVisible, handlePress }: HeaderActionRightProps) => {
+  const openFilterModal = () => {
+    // TODO: open filter modal functionality
+  };
```

Comment:
The linter is complaining about this unused `openFilterModal` function. We should remove it for now.